Makefile

```diff
@@ -22,12 +22,25 @@ build-local:
 build-test:
 	docker compose build pdf-service-test
 
-test-image: setup-directories
-	docker run --rm -d --name pdf-service pdf-service:latest-amd64
+test-image: setup-directories start-container run-inspec stop-container
+
+load-test-image: setup-directories start-container run-load stop-container
+
+start-container:
+	docker run --cpus=0.5 --memory=1G --rm -d --name pdf-service pdf-service:latest-amd64
 	sleep 2
-	inspec exec inspec -t docker://pdf-service --reporter cli junit:test-results/junit/pdf-service-inspec.xml
+
+stop-container:
 	docker container kill pdf-service
 
+run-inspec:
+	inspec exec inspec -t docker://pdf-service --reporter cli junit:test-results/junit/pdf-service-inspec.xml
```

Comment:
The PDF generated in this is tiny, so might not be representative, but probably good enough to catch it falling over on traffic alone. Did you try it locally?